antlr4ide.core/src/main/java/com/github/jknack/antlr4ide/generator/Antlr4Generator.xtend

```diff
@@ -51,6 +52,8 @@ class Antlr4Generator implements IGenerator {
    * @param fsa The Xtext file system access (not used).
    */
   override doGenerate(Resource resource, IFileSystemAccess fsa) {
+  if(DEBUG) System::out.println("Antlr4Generator doGenerate #0. optionsProvider.class >"+optionsProvider.getClass()+"<")
```

Comment:
I have to say, that even though I do not completely understand why xtend is needed, it is pretty impressive what this antlr4ide is capable of.